### ISSUE TITLE
Does someone succed to run ardour for live recording with overwitch ?

### ISSUE THREAD

**Issue by Sercurio** [dagargo/overwitch#39]
I'm trying to record and apply fx on individual tracks like Ableton, but with Ardour and overwitch.
I'm a little lost for the configuration.
Can someone, help me ?

**Comment by dagargo** [comment]
Your question is quite open so I'll tell you how is my setup.

I use `qjackctl` to start JACK. In the options, you can configure an application to run **after** the server is started. I've put `overwitch &` there.

Then, I open Ardour and configure the Audio and MIDI connections like in the capture below. This is what I use and let me run my Digitakt in sync (via midi clock) and start and stop from Ardour works too. (You'll need to configure your devices to receive clock and transport).

![Captura de pantalla de 2022-09-05 10-04-31](https://user-images.githubusercontent.com/4937384/188399928-2fe97fe9-49cd-4ad8-b8cf-a57cbcf07b9d.png)

If you proceed like this, Ardour will remember all your connections in the session so opening a project will keep your configuration.

Basically, I start the server, then run Overwitch and then Ardour.

Does this work for you?

**Comment by dagargo** [comment]
Did you finally get it to work?

**Comment by dagargo** [comment]
Did you install the udev rules? This is from the `README.md`. 

> Some udev rules might need to be installed manually with sudo make install from the udev directory as they are not part of the install target. This is not needed when packaging or when distributions already provide them.

**Comment by dagargo** [comment]
Did you manage to run it without root permissions?
If so, can we close this issue as solved?

**Comment by decibyte** [comment]
FWIW: I was messing around with this last week and I definitely didn't have to `sudo` anything after building and installing stuff. I _did_ do the udev rules dance.

I don't have versions at hand atm, but it was a home built overwitch from master branch and most recent executable from the Ardour website.

Let me use this chance to also thank you for your work on integrating the Elektron (and other) devices into the open source world, dagargo. It's very much appreciated :heart:

**Comment by dagargo** [comment]
I'm closing this as not repro.

Feel free to reopen in case the issue persist.
